web/satellite/src/components/account/billing/monthlySummary/MonthlyBillingSummary.vue

```diff
@@ -96,6 +96,10 @@ export default class MonthlyBillingSummary extends Vue {
      * toggleUsageChargesPopup is used to open/close area with list of project charges.
      */
     public toggleUsageChargesPopup(): void {
+        if (this.usageCharges.length === 0) {
```

Comment:
if (!this.usageCharges.length)?

Reply:
IMO straight comparison is more readable in this case